be/src/olap/rowset/segment_v2/encoding_info.cpp

```diff
@@ -67,6 +68,18 @@ struct TypeEncodingTraits<type, RLE> {
     }
 };
 
+template<FieldType type>
+struct TypeEncodingTraits<type, DICT_ENCODING> {
+    static Status create_page_builder(const PageBuilderOptions& opts, PageBuilder** builder) {
+        *builder = new BinaryDictPageBuilder(opts);
+        return Status::OK();
```

Comment:
maybe this type function can return void

Reply:
agree ,it seems that there is no RETURN_IF_ERROR logic in class constructor;
And I think it's  better to make a new commit to fix it.